Dockerfile.ocp

```diff
@@ -5,31 +5,24 @@ FROM ironic-builder AS builder
 # much as possible
 
 COPY ./build-ipa.sh /usr/local/bin/build-ipa.sh
+COPY ./chrooted.sh /usr/local/bin/chrooted.sh
 
 RUN dnf upgrade -y && \
-    dnf install -y \
-      cpio \
-      dracut \
-      && \
-    dnf download --destdir /tmp/packages \
-      openstack-ironic-python-agent \
-      python3-bcrypt \
-      python3-ironic-lib \
-      python3-ironic-python-agent \
-      rhosp-director-images-ipa-$(uname -m) \
-      && \
+    dnf install -y cpio dracut && \
```

Comment:
I split the command in different lines because it was easier to see which packages were installed, since now we only have 2 I'm not sure it makes sense anymore.